lib/tool_belt/cherry_picker.rb

```diff
@@ -92,7 +93,11 @@ def write_cherry_pick_log(picks, release)
     end
 
     def log_entry(pick)
-      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      if pick['bugzilla_id']
+        "#{pick['bugzilla_id']} - #{pick['id']} : [#{pick['revision']}] #{pick['subject']}"
+      else
+        "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      end
```

Comment:
Checkout: http://projects.theforeman.org/issues/13166.json

If you look in the custom fields section you see the Bugzilla data that comes from Redmine.